Jenkinsfile.release

```diff
@@ -74,85 +78,95 @@ podTemplate(cloud: 'openshift', label: pod_label, yaml: pod) {
-            stage('GCP: Image Promotion') {
-                shwrap("""
-                # pick up the project to use from the config
-                gcp_project=\$(jq -r .project_id \${GCP_IMAGE_UPLOAD_CONFIG})
-                ore gcloud promote-image \
-                    --log-level=INFO \
-                    --project=\${gcp_project} \
-                    --json-key \${GCP_IMAGE_UPLOAD_CONFIG} \
-                    --family fedora-coreos-${params.STREAM} \
-                    --image "${gcp_image}"
-                """)
+            // For production streams, promote the GCP image so that it
+            // will be the chosen image in an image family and deprecate
+            // all others. `ore gcloud promote-image` does this for us.
+            if ((basearch == 'x86_64') && params.STREAM in streams.production) {
+                stage('GCP ${basearch}: Image Promotion') {
+                    if (meta.gcp?.image) {
+                        gcp_image = meta.gcp.image
+                        shwrap("""
+                        # pick up the project to use from the config
+                        gcp_project=\$(jq -r .project_id \${GCP_IMAGE_UPLOAD_CONFIG})
+                        ore gcloud promote-image \
+                            --log-level=INFO \
+                            --project=\${gcp_project} \
+                            --json-key \${GCP_IMAGE_UPLOAD_CONFIG} \
+                            --family fedora-coreos-${params.STREAM} \
+                            --image "${gcp_image}"
+                        """)
+                    }
+                }
             }
-        }
 
-        if (params.AWS_REPLICATION == 'true') {
-            // Replicate the newly uploaded AMI to other regions. Intentionally
-            // split out from the 'Upload AWS' stage to allow for tests to be added
-            // at a later date before replicating said image.
-            //
-            // We have to re-run the coreos-meta-translator as aws-replicate
-            // only modifies the meta.json
-            stage('Replicate AWS AMI') {
-                shwrap("""
-                export AWS_CONFIG_FILE=\${AWS_FCOS_BUILDS_BOT_CONFIG}
-                cosa aws-replicate --build=${params.VERSION} --log-level=INFO
-                cosa generate-release-meta --build-id ${params.VERSION} --workdir .
-                cosa buildupload --build=${params.VERSION} --skip-builds-json s3 --acl=public-read ${s3_stream_dir}/builds
-                """)
+
+            if ((basearch in ['x86_64', 'aarch64']) && params.AWS_REPLICATION == 'true') {
+                // Replicate the newly uploaded AMI to other regions. Intentionally
+                // split out from the 'Upload AWS' stage to allow for tests to be added
+                // at a later date before replicating said image.
+                //
+                // We have to re-run the coreos-meta-translator as aws-replicate
+                // only modifies the meta.json
```

Comment:
Minor: should move that comment down.

Reply:
hmm - I don't see where we run coreos-meta-translator at all - old comment?

Reply:
Yeah, old name for `cosa generate-release-meta`. I think the comment is still valuable. I meant moving it down to where we run `cosa generate-release-meta` now (and I guess update the comment too).

Reply:
+1 added an updated comment below